### PR TITLE
New package: tdacli-0.3.0

### PR DESCRIPTION
**Pull request by jason1987d**
#### Testing the changes
- I tested the changes in this PR: YES

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): YES

### CODE REVIEW

File: srcpkgs/tdacli/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+pkgname=tdacli
+version=0.3.0
+revision=1
+build_style=cargo
+makedepends="pkg-config openssl-devel"
+short_desc="TDAmeritrade API client"
+maintainer="Jason Manley <jason@jasondavid.tv>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/jbertovic/tdacli/"
+distfiles="https://codeload.github.com/jbertovic/${pkgname}/tar.gz/refs/tags/v${version}>${pkgname}-${version}.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/tdacli/template:11]: you shouldn't need to rename the file, and you should use the pre-redirected url:

```suggestion
distfiles="https://github.com/jbertovic/tdacli/archive/refs/tags/v${version}.tar.gz"
```

File: srcpkgs/tdacli/template [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'tdacli'
+pkgname=tdacli
+version=0.3.0
+revision=1
+build_style=cargo
+makedepends="pkg-config openssl-devel"
+short_desc="TDAmeritrade API client"
+maintainer="Jason Manley <jason@jasondavid.tv>"
+license="GPL-3.0-or-later"
```
Comment (classabbyamp) [inline srcpkgs/tdacli/template:9]: this is not the correct licence

**Comment by jason1987d** [comment]
I think makedepends= should just be hostmakedepends= , right? Currently testing with both lines, I think they are redundant though.

**Comment by classabbyamp** [comment]
pkg-config should be hostmake, openssl-devel should be make

**Comment by jason1987d** [comment]
Attempt to bump. I use this package all the time, has not had more recent updates.

**Comment by jason1987d** [comment]
I am fine with this being closed. TDAmeritrade API is deprecated.

**Pull request closed without merging**